ETH-Merge-Bots/Milestones-in-TTD/src/utils.ts

```diff
@@ -0,0 +1,58 @@
+import BigNumber from "bignumber.js";
+
+export const getNumberOfDays = (avgBlockTime: number, estimatedNumberOfBlocksUntilMerge: BigNumber) => {
+  // One day in milliseconds
+  const oneDay = 1000 * 60 * 60 * 24;
+
+  const date = new Date();
+  date.setSeconds(estimatedNumberOfBlocksUntilMerge.multipliedBy(avgBlockTime).plus(date.getSeconds()).toNumber());
+
+  const estimatedMergeDate = new Date(date);
+
+  const now = new Date(Date.now());
```

Comment:
Could you have used  `const estimatedMergeDate = date` and  `const now = new Date()`? (genuine question xD)